src/connector/src/lib.rs

```diff
@@ -43,6 +43,10 @@ use duration_str::parse_std;
 use serde::de;
 
 pub mod aws_utils;
+
+#[rustfmt::skip]
+pub mod changeable_fields;
```

Comment:
In rust, it seems better to name with `mutable`.

Reply:
+1 for `mutable` (or maybe `alterable`)

However, there's still an ambiguity: alterable on the fly vs alterable via rebuild

Reply:
prefer `allow_alter_on_fly`